cores/esp8266/Tone.cpp

```diff
@@ -70,6 +70,12 @@ void tone(uint8_t _pin, double frequency, unsigned long duration) {
 }
 
 
+// Fix ambiguous tone() binding when adding in a duration
+void tone(uint8_t _pin, int frequency, unsigned long duration) {
+  tone(_pin, frequency, duration);
```

Comment:
Correct me, I think this one is calling itself.

Reply:
D'oh!  I've been away from the keyboard too long.  Will fix that.